### ISSUE TITLE
"default host resolves to multiple instances"

### ISSUE THREAD

**Issue by devsecops-pe** [hamnsk/vmware-exporter#6]
Hi, 
When we try to open the web page using probe:
(http://<ip_prom_exporter>:<port>/probe?target=<vcenter_server_url>)
we receive the error in the object.

Could you help us?
Thank you

**Comment by hamnsk** [comment]
No, I can't help you because I don't have enough information. Working with vCenter is not guaranteed, I don't have it, I can't check.
